### ISSUE TITLE
[BUG] No module named 'yaml' after startup

### ISSUE THREAD

**Issue by ntlhui** [UCSD-E4E/radio_collar_tracker_dsp2#5]
At commit c7279792c2f841b07f26f3eb23eae85028195ac6

```
$ uname -a
Linux e4e-upcore-testbench 5.4.0-131-generic #147~18.04.1-Ubuntu SMP Sat Oct 15 13:10:18 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

```
$ sudo service rctrun status
● rctrun.service - RCT OBC Runner
   Loaded: loaded (/lib/systemd/system/rctrun.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Tue 2022-11-01 21:48:15 PDT; 15h ago
 Main PID: 694 (code=exited, status=1/FAILURE)

Nov 01 21:48:15 e4e-upcore-testbench rctrun[694]:     return ep.load()
Nov 01 21:48:15 e4e-upcore-testbench rctrun[694]:   File "/home/e4e/radio_collar_tracker_dsp2/.venv/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2324, in load
Nov 01 21:48:15 e4e-upcore-testbench rctrun[694]:     return self.resolve()
Nov 01 21:48:15 e4e-upcore-testbench rctrun[694]:   File "/home/e4e/radio_collar_tracker_dsp2/.venv/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2330, in resolve
Nov 01 21:48:15 e4e-upcore-testbench rctrun[694]:     module = __import__(self.module_name, fromlist=['__name__'], level=0)
Nov 01 21:48:15 e4e-upcore-testbench rctrun[694]:   File "/home/e4e/radio_collar_tracker_dsp2/.venv/lib/python3.6/site-packages/autostart/rctrun.py", line 15, in <module>
Nov 01 21:48:15 e4e-upcore-testbench rctrun[694]:     import yaml
Nov 01 21:48:15 e4e-upcore-testbench rctrun[694]: ModuleNotFoundError: No module named 'yaml'
Nov 01 21:48:15 e4e-upcore-testbench systemd[1]: rctrun.service: Main process exited, code=exited, status=1/FAILURE
Nov 01 21:48:15 e4e-upcore-testbench systemd[1]: rctrun.service: Failed with result 'exit-code'.
```

**Comment by ntlhui** [comment]
We probably need to add `yaml` to the `install_requires` in `setup.py`

**Comment by ntlhui** [comment]
Fixed in 862b7cebe2847c21d0876d61c5482d2566acd6f9
